packages/requisitions/src/utils.ts

```diff
@@ -3,15 +3,21 @@ import {
   LocaleKey,
   useTranslation,
 } from '@openmsupply-client/common';
-import { Requisition } from './types';
 import { ResponseRequisitionRowFragment } from './ResponseRequisition/api';
+import { RequestRequisitionRowFragment } from './RequestRequisition/api';
```

Comment:
misc cleaning up of this file. Think I'm happy with it in general, though there is a lot of duplication and it could be more DRY I don't think it's worthwhile